dubbo-config/dubbo-config-api/src/test/java/org/apache/dubbo/config/integration/multiple/exportmetadata/MultipleRegistryCenterExportMetadataIntegrationTest.java

```diff
@@ -99,13 +91,9 @@ public void setUp() throws Exception {
         DubboBootstrap.getInstance()
             .application(new ApplicationConfig(PROVIDER_APPLICATION_NAME))
             .protocol(new ProtocolConfig(PROTOCOL_NAME))
-            .service(serviceConfig);
-        for (RegistryCenter.Instance instance : registryCenter.getRegistryCenterInstance()) {
-            DubboBootstrap.getInstance().registry(new RegistryConfig(String.format("%s://%s:%s",
-                instance.getType(),
-                instance.getHostname(),
-                instance.getPort())));
-        }
+            .service(serviceConfig)
+            .registry(new RegistryConfig("zookeeper://127.0.0.1:2181"))
+            .registry(new RegistryConfig("zookeeper://127.0.0.1:2182"));
```

Comment:
Is it possible to define global zk server port/address variables and then quote them in test? It is better not to use the port 2181 by default to avoid port conflicts on the zk server. Just a suggestion.

Reply:
That's a good seggustion! However, all local `zooKeeper connection strings` aren't displayed clearly, especially using Spring configuration or annotation.

But I've supported this feature.

Reply:
The best way is to define two client ports (which might not be `2181` and `2182`) and quote them when using. However, there are so many hard code in unit test. So, it's very diffcult to change anywhere.

such as some zookeeper connection string in properties file

```java
dubbo.registry.address=zookeeper://127.0.0.1:2181?registry-type=service
biz.group=greeting
biz.group2=group2
dubbo.call-timeout=2000
```